ammonia-CO-H2-Alzueta-2023.yaml

```diff
@@ -0,0 +1,2549 @@
+  - name: N2
+    eps: {A: 7.56771e+00, b: -1.96878e-01, Ea: 2.53168e+02}
+  - name: H2O
+    eps: {A: 3.35451e+00, b: 1.33352e-01, Ea: 6.64633e+01}
+  note: |-
+    Ar is the reference collider 'M'.
+    P. J. Singal, J. Lee, L. Lei, R. L. Speth, M. P. Burke, Implementation of New Mixture Rules Has a Substantial Impact on Combustion Predictions for H2 and NH3, Proc. Combust. Inst. 40 (2024).
+    A. W. Jasper, Predicting third-body collision efficiencies for water and other polyatomic baths. Faraday Discuss. 238 (2022) 68-86.
+
+common-reactions:
+- equation: H + O2 <=> O + OH
+  rate-constant: {A: 1.0e+14, b: 0.0, Ea: 1.5286e+04}
+  note: |-
+    *****************************************************************************
+       H2/O2 subset                                                             *
+    *****************************************************************************
+- equation: O + H2 <=> OH + H
+  duplicate: true
+  rate-constant: {A: 3.8e+12, b: 0.0, Ea: 7948.0}
+  note: Hong et al., Proc. Comb. Inst. 33:309-316 (2011)
+- equation: O + H2 <=> OH + H
+  duplicate: true
+  rate-constant: {A: 8.8e+14, b: 0.0, Ea: 1.9175e+04}
+- equation: OH + H2 <=> H + H2O
+  rate-constant: {A: 2.2e+08, b: 1.51, Ea: 3430.0}
+  note: Baulch DL Bowman CT Cobos CJ Cox RA Just Th Kerr JA Pilling MJ Stocker
+    D Troe J Tsang W Walker RW Warnatz J JPCRD 34:757-1397 2005
+- equation: OH + OH <=> O + H2O
+  duplicate: true
+  rate-constant: {A: 2.0e+07, b: 1.651, Ea: 631.0}
+  note: |-
+    Michael and Sutherland, J. Phys. Chem. 92:3853 (1988)
+     M. Sangwan, L. N. Krasnoperov, J Phys Chem A 116 (2012) 1181711822.
+- equation: OH + OH <=> O + H2O
+  duplicate: true
+  rate-constant: {A: 2.6e+11, b: -0.057, Ea: -827.0}
+- equation: H2 + M <=> H + H + M
+  type: three-body
+  rate-constant: {A: 4.6e+19, b: -1.4, Ea: 1.0438e+05}
+  efficiencies: {H2: 2.5, H2O: 12.0, AR: 0.0, O2: 0.0}
+  note: |-
+    HH: Refitted (400-3000 K) for better stability of the solvers (avoiding
+    negative 'A') 16-04-08"
+- equation: H2 + AR <=> H + H + AR
+  rate-constant: {A: 5.8e+18, b: -1.1, Ea: 1.0438e+05}
+- equation: H + O + M <=> OH + M
+  type: three-body
+  rate-constant: {A: 4.7e+18, b: -1.0, Ea: 0.0}
+  efficiencies: {H2: 2.5, H2O: 12.0, AR: 0.75, CO: 1.9, CO2: 3.8, O2: 0.0}
+  note: Tsang and Hampson, J. Phys. Chem. Ref. Data, 15:1087 (1986)
+- equation: O + O + M <=> O2 + M
+  type: three-body
+  rate-constant: {A: 1.9e+13, b: 0.0, Ea: -1788.0}
+  efficiencies: {H2: 2.5, H2O: 12.0, AR: 0.0, CO: 1.9, CO2: 3.8}
+  note: Tsang and Hampson, J. Phys. Chem. Ref. Data, 15:1087 (1986)
+- equation: H2O + H2O <=> H + OH + H2O
+  rate-constant: {A: 1.0e+26, b: -2.44, Ea: 1.2018e+05}
+  note: |-
+    Srinivasan and Michael, Int. J. Chem. Kinetic. 38 (2006)
+    Rate constant is for Ar with efficiencies from Michael et al., J. Phys. Chem. A, 106 (2002)
+    Efficiencies for CO and CO2 taken from Li et al., Int. J. Chem. Kinet. 36:566-575 (2004)
```

Comment:
Unfortunately, I think the original mechanism is written such that the comment relevant to each reaction occurs on the lines following that reaction, whereas `ck2yaml` assumes that the comment for a reaction comes first. This means that all of the `note`s here are associated with the wrong reactions.